floodlight/core/events.py

```diff
@@ -61,3 +95,148 @@ def custom(self):
         )
 
         return custom
+
+    @property
+    def essential_missing(self):
+        missing_columns = [
+            col for col in essential_events_columns if col not in self.essential
+        ]
+
+        if not missing_columns:
+            return None
+        else:
+            return missing_columns
```

Comment:
```suggestion
        if not missing_columns:
            missing_columns = None
        
        return missing_columns
```

Reply:
I feel this is an redundant else clause and double return, or am I missing something? Applies for all properties